workflow-bot-app/src/test/java/com/symphony/bdk/workflow/WorkflowValidatorTest.java

```diff
@@ -47,14 +43,6 @@ static Stream<Arguments> firstActivities() {
     );
   }
 
-  @ParameterizedTest
-  @MethodSource("firstActivities")
```

Comment:
Either delete the stream firstActivities() as it is not used anymore,  or keep it and make the test to pass instead of failing, to cover this case where starting activity has a condition

Reply:
removed